src/screens/Documents/View/index.js

```diff
@@ -87,16 +87,22 @@ class DocumentsViewScreen extends NavigatorComponent<Props & DocumentsState & Ac
     }
   }
 
+  dismissModal = () => {
+    this.setState({ moreMenuVisible: false });
+  }
+
   onSelectEdit = () => {
     const { openedDocumentId } = this.props;
     if (openedDocumentId == null) return;
     const document = getOpenedDocument(this.props);
     if (document == null) return;
-
     this.props.startDocumentEditing(openedDocumentId);
     this.props.navigator.showModal({
       ...screen('DOCUMENT_MODIFY_SCREEN'),
       title: document.name,
+      passProps: {
+        onCancel: () => this.dismissModal(),
```

Comment:
There is no need to use fat arrow functions twice. You can just pass `this.dismissModal` here.

Reply:
Done